scifight/models.py

```diff
@@ -16,6 +20,16 @@
     hold a number plus possible brief one-word explanation. """
 
 
+class Tournament(models.Model):
+    name        = models.CharField(max_length=NAME_LENGTH)
+    slug        = models.SlugField(max_length=SLUG_LENGTH, unique=True)
+    description = models.TextField(blank=True, null=True)
+    create_data = models.DateTimeField(default=timezone.now)
```

Comment:
Do we really want the *database default* for this field?
How about the following:
```
class Tournament(models.Model):
    name        = models.CharField(max_length=NAME_LENGTH)
    slug        = models.SlugField(max_length=SLUG_LENGTH, unique=True)
    description = models.TextField(blank=True, null=True)
    opening_date = models.DateTimeField(initial=timezone.now)
    closing_date = models.DateTimeField(initial=timezone.now)
```
Please also note that «data» and «date» are different words.